app/controllers/sign_and_submit_controller.rb

```diff
@@ -6,10 +6,32 @@ def update
 
     if @step.valid?
       current_app.update(step_params)
+      create_dhs1171_pdf
       flash[:notice] = "Your application has been successfully submitted."
       redirect_to root_path(anchor: "fold")
     else
       render :edit
     end
   end
+
+  private
+
+  def create_dhs1171_pdf
```

Comment:
yes totally agree! When I was writing the controller specs for this it was a total pain and I wanted to just have an object to stub. But I think we should take care of it separately / when we start adding more fields.